Exercise_D.js

```diff
@@ -15,6 +15,14 @@ Use a combination of conditionals and loops to create your answer for this quest
 
 function multiplyNumber(number) {
   // CODE HERE
+if(number < 1 || typeof number !== "number"){
+  return "ERROR";
+}else if(number >= 1000000){
+  return number;
+}else{
+  number *= 10
+  return number;
+}
 }
 
 console.log(multiplyNumber(10));

```

Comment:
For this question, you would need to make use of a while loop at the end because if the number is smaller than 1000000, we would need to use a do while loop to calculate how many times it multiplies to get the number from whichever number it is to 1000000.